### PR TITLE
Unit test framework

### PR DESCRIPTION
**Pull request by randres1640**

### CODE REVIEW

File: spec/api/spec_helper.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+  
+ENV["RACK_ENV"] = "test"
+require_relative '../../api/app'
+raise "test database doesn't contain test" unless DB.opts[:database] =~ /test/
```
Comment (doggygit) [inline spec/api/spec_helper.rb:4]: :thinking: maybe write `"database name doesn't contain test"` instead.

File: spec/minitest_helper.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+ENV['MT_NO_PLUGINS'] = '1' # Work around stupid autoloading of plugins
+gem 'minitest'
```
Comment (doggygit) [inline spec/minitest_helper.rb:2]: OK, I have never seen such a thing outside a `Gemfile`

Please remove or educate me what it is doing :)

Comment (hendrikb) [reply spec/minitest_helper.rb:2]: Same for me, shouldn't we try to keep gem loading instructions inside `Gemfile` work with something like

```ruby 
group :test do
  gem 'minitest'
end
```

**Comment by hendrikb** [comment]
@randres1640 can you also please add instructions to the `README.md` how to run the tests quickly?

**Pull request merged**